app/jobs/hyrax/grant_edit_to_members_job.rb

```diff
@@ -1,15 +1,16 @@
 module Hyrax
   # Grants edit access for the supplied user for the members attached to a work
-  class GrantEditToMembersJob < ApplicationJob
+  class GrantEditToMembersJob <  Hyrax::ApplicationJob
```

Comment:
```suggestion
  class GrantEditToMembersJob < ApplicationJob
```
this ought to be fine. `Hyrax` is implied since this is the module namespace for this job.